OSPSuite.Presentation.Importer/Core/DataSourceFileReaders/ExcelDataSourceFile.cs

```diff
@@ -0,0 +1,62 @@
+﻿using System;
+using System.Collections.Generic;
+using OSPSuite.Core.Services;
+using OSPSuite.Infrastructure.Import.Services;
+using OSPSuite.Presentation.Importer.Infrastructure;
+
+namespace OSPSuite.Presentation.Importer.Core.DataSourceFileReaders
+{
+   public interface IExcelDataSourceFile : IDataSourceFile
+   {
+   }
+
+   public class ExcelDataSourceFile : DataSourceFile, IExcelDataSourceFile
+   {
+      public ExcelDataSourceFile(IImportLogger logger) : base(logger)
+      {
+      }
+
+      protected override Dictionary<string, IDataSheet> LoadFromFile(string path)
+      {
+         try
+         {
+            var loadedData = new Dictionary<string, IDataSheet>();
+
+
+            var reader = new ExcelReader(path);
+
+            while (reader.MoveToNextSheet())
+            {
+               var sheetName = reader.CurrentSheet.SheetName;
+               var headers = new List<string>();
+               IDataSheet dataSheet = new DataSheet();
+               dataSheet.RawData = new UnformattedData();
+
+               if (reader.MoveToNextRow())
+                  headers = reader.CurrentRow;
+               //else probably throw exception
```

Comment:
to throw or not to throw :)(

Reply:
fixed it so if we get a totally empty sheet we just ignore it